Message/LoginViewController.swift

```diff
@@ -8,16 +8,15 @@
 
 import UIKit
 import FirebaseAuth
-import Kingfisher
 import Validator
+import Then
+import Reusable
 
 private struct CheckCharater {
     let inputEmpty: Int = 1
-    let emailMaxLength: Int = 32
-    let passwordMinLength: Int = 7
 }
 
-class ViewController: UIViewController {
+class LoginViewController: UIViewController, StoryboardSceneBased {
```

Comment:
```suggestion
final class LoginViewController: UIViewController {
```
conform StoryboardSceneBased thì để hẳn ra extension nhé